server/app/scanpy_engine/scanpy_engine.py

```diff
@@ -50,41 +50,58 @@ def _get_default_config():
             "diffexp_lfc_cutoff": 0.01,
         }
 
-    def _alias_annotation_names(self, axis, name):
+    @staticmethod
+    def _create_unique_column_name(df, prefix):
+        """ given a dataframe and a name prefix, return a column name which does not
+            exist in the dataframe.
```

Comment:
Can you add that it does this by appending a number?

Reply:
done.